packages/fxa-payments-server/server/config/index.js

```diff
@@ -17,6 +17,12 @@ const conf = convict({
       env: 'SUBSCRIPTIONS_FIRESTORE_CONFIGS_ENABLED',
       format: Boolean,
     },
+    useStripeAutomaticTax: {
+      default: false,
+      doc: 'Feature flag on whether Stripe automatic tax is enabled',
+      env: 'SUBSCRIPTIONS_STRIPE_TAX_ENABLED',
+      format: Boolean,
+    },
```

Comment:
I thought this would be entirely a back-end thing?

Reply:
@bbangert as discussed on Slack, looks like we do need a front-end change. Does this unblock the PR?

Reply:
Yep, looks fine!